### DISCUSSION TITLE
How do I delete the NPM cache ? I have it running on Unraid

### DISCUSSION THREAD

**Discussion by Northwave** [jlesage/docker-nginx-proxy-manager#333] [Q&A] [answered]
As the title already tells, i need to know how I can delete the website cache of NPM. I have it running on Unraid if that helps

**Comment by jlesage** [comment]
Sorry, not sure to see which cache you are talking about.

**Comment by Northwave** [comment]
Hi, I mean the cache of a proxy host.

**Reply by jlesage** [reply, marked as answer]
It seems that caching is not enabled, unless `Cache Assets` is enabled for the proxy host. This will cache files like images, css, js, etc. Cached files should be stored under `/var/lib/nginx/cache/public` inside the container.

**Reply by Northwave** [reply]
ok thank you. that should solve my issue as some outdated js is used for a proxy host were it was enabled to cache assets
